src/Compiler/Driver/ParseAndCheckInputs.fs

```diff
@@ -1288,7 +1268,7 @@ let CheckOneInputAux
                             tcState
```

Comment:
The rootSig was already being added to the tcState here.  We shouldn't have been adding it again in AddDummyCheckResultsToTcState